portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase

```diff
@@ -567,7 +567,7 @@ definition {
 
 			ACDXPSettings.goToInstanceSettingsAC();
 
-			ACDXPSettings.connectAnalyticsCloud();
```

Comment:
Hey @joshitagaki, `Site Name` is not created in this test, so we should use `ACDXPSettings.connectAnalyticsCloud();`

Reply:
Hey @ViniLopes20 , using the `ACDXPSettings.connectAnalyticsCloud()` macro was causing issues on CI where AC wasn't creating a property in time of the test so the steps following would fail. Finishing the setup gives the CI enough time to finish loading of AC. Here is an example run I did of this and the reason I made that change: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=2249358495&testrayRunId=2249359683. It doesn't change or affect the outcome of the test. Let me know if you think that makes sense. Thanks!

Reply:
Ohhh yeah, you're right. I put the wrong macro name in my comment. You could just use `ACDXPSettings.connectDXPtoAnalyticsCloud();` because you will be syncing from the Liferay DXP site. You don't have the Site Name created, so you don't need to pass the site name that doesn't exist in your test run

But I don't know why using `ACDXPSettings.connectAnalyticsCloud();` didn't create the property if a property is automatically created after the connection is made, but I think you could use `ACDXPSettings.connectDXPtoAnalyticsCloud();` for that case

Reply:
Ahh ok true we can do that instead of passing the site name.

I think it's just a timing issue, because there was a test passing with this macro but it seems to be very flaky with the property creation in time.

Reply:
Yeah probably, thanks josh for noting and explaining this 😄